README.md

```diff
@@ -1,24 +1,185 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Blog App <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**Blog App** is a project aimed at creating a comprehensive and dynamic platform for bloggers and readers alike. With an emphasis on seamless user experience and content engagement, this application is designed to empower writers to share their thoughts, stories, and insights with the world. Through a user-friendly interface, bloggers can easily compose and publish articles, while readers can explore a diverse range of topics and engage in meaningful discussions.
 
-* Deployment instructions
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User-Friendly Writing:** Easily compose and publish articles using a straightforward and intuitive writing interface. Create, edit, and share your thoughts effortlessly.
+
+- **Engage with Comments:** Interact with your readers through a seamless commenting system. Receive feedback, answer questions, and foster a sense of community around your posts.
+
+- **Effortless Discovery:** Discover new content with ease using the app's user-friendly search feature. Find articles, authors, and topics that interest you in no time.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Non deployed yet
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need:
+
+
+
+```sh
+ gem install rails
+```
+
+**Setup**
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+**Install**
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+**Usage**
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+**Run tests**
+To run tests, run the following command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+
+**Deployment**
+You can deploy this project using:
```

Comment:
### Optional
- [ ] It is important to fill all the sections correctly so that the README file will be professional and would also be easy for non-developers to have a quick view of the app